docs/book/src/developer/tilt.md

```diff
@@ -35,19 +35,23 @@ You can see the status of the cluster with:
 kubectl cluster-info --context kind-capi-test
 ```
 
-### Create a tilt-settings.json file
-
-Next, create a `tilt-settings.json` file and place it in your local copy of `cluster-api`. Here is an example:
-
-```json
-{
-  "default_registry": "gcr.io/your-project-name-here",
-  "provider_repos": ["../cluster-api-provider-aws"],
-  "enable_providers": ["aws", "docker", "kubeadm-bootstrap", "kubeadm-control-plane"]
+### Create a tilt-settings.yaml file
+
+Next, create a `tilt-settings.yaml` file and place it in your local copy of `cluster-api`. Here is an example:
+
+```yaml
+default_registry: gcr.io/your-project-name-here
+provider_repos:
+- ../cluster-api-provider-aws
+enable_providers": 
+- aws
+- docker
+- kubeadm-bootstrap
+- kubeadm-control-plane
 }
 ```
 
-#### tilt-settings.json fields
+#### tilt-settings.yaml fields
```

Comment:
```suggestion
#### tilt-settings.{json|yaml} fields
```

Reply:
same as above, let's drop the extension and keep things simple